docusaurus/docs/getting-started/how-to-use.md

```diff
@@ -4,4 +4,13 @@ title: 'How to Use CXXGraph'
 tags: ['introduction', 'how to use', 'getting started']
 ---
 ### How do I put CXXGraph on my project?
-To use the library simply **put the header file where you need it.** It’s that easy!
\ No newline at end of file
+
+To use CXXGraph in your project, simply put the header file `CXXGraph.hpp` where you need it, and include the following in your header:
+
+```
+include "CXXGraph.hpp"
+
+using namespace CXXGRAPH;
+```
+
+It's that easy!
```

Comment:
Sadly, not yet! We haven't created a deployment/combined header that can be simply included into a project. I think @ZigRazor is tracking some work to add that.

Reply:
This is true after the installation, @nolankramer  can you open an issue to track this missing, or almost misconfigured, installation phase in cmake? 
Thank you in advance, I think in any case that this pull request can be merged, you agree?

Reply:
I have opened the new Issue

Reply:
If we update the instructions here to specify using CPM (which I use in my own project), this should be mergeable.

For now, using CPM looks like this:
```
CPMAddPackage("gh:ZigRazor/CXXGraph")
if (CXXGraph_ADDED)
    add_library(CXXGraph INTERFACE IMPORTED GLOBAL)
    target_include_directories(CXXGraph INTERFACE "${CXXGraph_SOURCE_DIR}/include")
endif()
```

Reply:
If Zigrazor wants this in the how to use doc I'll gladly add a section.